merino/config_sentry.py

```diff
@@ -22,9 +28,36 @@ def configure_sentry() -> None:  # pragma: no cover
         ],
         release=version_sha,
         debug="debug" == settings.sentry.mode,
+        before_send=strip_sensitive_data,
         environment=settings.sentry.env,
         # Set traces_sample_rate to 1.0 to capture 100%
         # of transactions for performance monitoring.
         # We recommend adjusting this value in production,
         traces_sample_rate=settings.sentry.traces_sample_rate,
     )
+
+
+def strip_sensitive_data(event: dict, hint: dict) -> Any:
+    """Filter out sensitive data from Sentry events."""
+    #  See: https://docs.sentry.io/platforms/python/configuration/filtering/
+    if event["request"]["query_string"]:
+        event["request"]["query_string"] = ""
+
+    if "exc_info" in hint:
+        exc_type, exc_value, tb = hint["exc_info"]
+        if isinstance(exc_value, RuntimeError) or isinstance(exc_value, BackendError):
```

Comment:
```suggestion
        if isinstance(exc_value, (RuntimeError, BackendError)):
```
BTW, I don't think we have explicitly raised any `RuntimeError` in Merino, shall we just do this for all exceptions?

Reply:
Correct, therefore I think that's sensible!  Many child exceptions fall under `RuntimeError`, but capturing all under `Exception` is a good idea.